Lib/fontTools/misc/psCharStrings.py

```diff
@@ -164,6 +170,14 @@ def buildOperatorDict(operatorList):
 	((12, 37),	'flex1'),
 ]
 
+CFF2Operators = [entry for entry in t2Operators if ((isinstance(entry[0], int) or
+											(entry[0][1] < 34)) and (not entry[0] in [11,14])) ]
+CFF2Operators.extend(
+[
+	(15,		'vsindex'),
+	(16,		'blend'),
+])
+CFF2Operators.sort()
```

Comment:
this fails in Python 3 as you are comparing `int` and `tuple`.
https://ci.appveyor.com/project/fonttools/fonttools/build/1.0.123/job/i4k20he0fk551xat#L72